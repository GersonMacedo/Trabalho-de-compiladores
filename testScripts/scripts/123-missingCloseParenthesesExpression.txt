! -s SYNTACTIC
! should complain about missing )
program p ;
begin
    a := (true;
end .