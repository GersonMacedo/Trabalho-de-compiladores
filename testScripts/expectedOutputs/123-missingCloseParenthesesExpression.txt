
Performing up to syntactic analysis at testScripts/scripts/123-missingCloseParenthesesExpression.txt
Parser::Expected ')' but ';' was found at line 5 column 15
